### DISCUSSION TITLE
Cannot preview/download fies from console

### DISCUSSION THREAD

**Discussion by jpacora** [minio/minio#20088] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior
Get metadata, preview and download file using MinIO console using root account

## Current Behavior
Metadata, preview and download fetuares isn't working on a fresh docker container setup, but `mc get` is working
`mc get "rpi/prue/Screenshot 2024-04-28 at 19.58.10.png" local.png`
![image (2)](https://github.com/user-attachments/assets/d2e77b16-681e-48dd-b77a-20c9402f3972)
![image (3)](https://github.com/user-attachments/assets/429a1662-e785-459f-8bee-bf1c10e64688)

## Steps to Reproduce (for bugs)

1. Setup docker
2. Use the `minio/minio:latest` image from docker hub
3. Start with `server --console-address ":9001" /data` and ["MINIO_ROOT_USER", "MINIO_ROOT_PASSWORD", "MINIO_SERVER_URL", "MINIO_BROWSER_REDIRECT_URL"] env vars

## Your Environment

* Version used (`minio --version`):
* Server setup and configuration: Raspberry Pi 8GB RAM
* Operating System and version (`uname -a`): Linux raspberrypi 6.6.31+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.31-1+rpt1 (2024-05-29) aarch64 GNU/Linux

**Comment by harshavardhana** [comment, marked as answer]
For previews to work set the correct content-type for all the objects

**Reply by jpacora** [reply]
But downloads and metadata still not working, the backend keeps triggering error: Access Denied

**Reply by harshavardhana** [reply]
Then open an issue at https://github.com/minio/console/issues

**Reply by harshavardhana** [reply]
Use the exact version tag for the container while reporting the issue RELEASE.2024-07-13T01-46-15Z

**Reply by jpacora** [reply]
Ok

**Reply by jpacora** [reply]
@harshavardhana after a some hours of troubleshooting, I've realized that maybe CloudFlare is blocking the requests coming from the Console? Because I'm using a cloudflare tunnel to expose my raspberry pi to internet.

**Reply by harshavardhana** [reply]
Yeah if it caches yes.

**Reply by jpacora** [reply]
Thank you @harshavardhana , seems to CloudFlare cache was the problem (and was enabled by default), so after create a rule to disable cache at my subdomain now all is working!
